models/intermediate/int_jira__issue_versions.sql

```diff
@@ -0,0 +1,78 @@
+{{
+    config(
+        materialized='view')
+}}
+
+with version as (
+
+    select *
+    from {{ var('version') }}
+),
+
+version_history as (
+
+    select *
+    from {{ ref('int_jira__issue_multiselect_history') }}
+
+    where field_id = 'versions'
+        or field_id = 'fixVersions'
```

Comment:
Can you explain these a bit more during standup? I want to make sure we aren't missing anything else.

Reply:
yeah these docs may help as well https://confluence.atlassian.com/jirakb/general-use-of-affect-version-s-and-fix-version-s-838428168.html